src/HelloIntellige.java

```diff
@@ -1,6 +1,11 @@
+import java.util.Scanner;
+
 public class HelloIntellige {
 
     public static void main(String[]args){
-        System.out.println("Hello there");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Hi again this is Mehedy:");
+        System.out.println("Hi NSW this is Mehedy:");
+        String message = scanner.nextLine();
```

Comment:
Please review code, change it the way that message input from cmd will be shown next line 
and close scanner at the last line.